examples/node/player.js

```diff
@@ -1,6 +1,6 @@
 const yargs = require('yargs')
 
-const { pipelines } = require('../../lib/index.node.js')
```

Comment:
These changes are not really necessary, or?

Reply:
I was unable to run the example player. However, it is not really related to this issue. Removing it